python/bottle/webappWithTests/ArmTemplates/windows-webapp-template.json

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
assuming you have verified this is the final template in master

Reply:
Yes. This template was pulled from latest master